tests/__init__.py

```diff
@@ -30,12 +30,11 @@
 Standard set of databases to run tests against
 """
 
-LOCATION = "/var/tmp/mdapi-tests/"
+LOCATION = "/var/tmp/mdapi-tests/"  # noqa : S108
```

Comment:
Ruff flags this for a good reason 😉 – read e.g. this for some background: https://systemd.io/TEMPORARY_DIRECTORIES/

I’ve skimmed the tests using this and have some ideas how this could be done differently.

Reply:
I will attempt adding my own implementation using the [`tempfile` library](https://docs.python.org/3/library/tempfile.html) so that you have something to cringe on early in the morning over a mug of coffee.

Reply:
😂